book.rb

```diff
@@ -0,0 +1,18 @@
+require_relative './person'
+require_relative './student'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental, :person
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(rents)
+    @rental.push(rents)
+    rents.book = self
+  end
+end
```

Comment:
- [ ] Kindly change what this method does. Just like what the other reviewer said, this method should return a new instance of the `Rental` class:

```ruby
def add_rental(person, date)
  Rental.new(date, self, person)
end
```
 

- [ ] Kindly create this method in the Person class also. 

If you remember, in the initialize method of the `Rental` class, both the book and person objects that are passed in as parameters are pushing the instance of the rental into their array attributes at creation:

```ruby
# these lines of codes
book.rental << self
person.rental << self
```
This way, each book object can keep track of all the people that rented it while each person object can keep track of all the books that were rented by that same person through their rental array attributes. 👍